java/ql/test/kotlin/library-tests/methods/diagnostics.expected

```diff
@@ -0,0 +1 @@
+| enumClass.kt:8:3:11:4 | Unrecognised class kind ENUM_ENTRY |
```

Comment:
Shouldn't this diagnostic be removed instead of added?

Reply:
Doh, yes, committed the wrong test output; fixed now.